app/views/shared/_header.html.erb

```diff
@@ -5,45 +5,45 @@
     <nav class="site-nav">
       <ul>
         <li>
-          <%= link_to "GitHub Education", 'https://education.github.com', target: '_blank' %>
+          <%= link_to t('views.shared.github_education'), 'https://education.github.com', target: '_blank' %>
```

Comment:
@CGA1123 GitHub Education is standard no matter what language. Let's revert this back so that we don't have to duplicate this for every language we add.